xamarin/Xamarin.Forms
# [UWP] BoxView using default BackgroundColor and GestureRecognizers

### Description of Change ###

This PR include changes to allow the `BoxView` gestures being recognized when BackgroundColor set to Default.
 
### Issues Resolved ### 

- fixes #12755 

### API Changes ###
 
 None

### Platforms Affected ### 

- UWP

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

![fix12755](https://user-images.githubusercontent.com/6755973/100625810-4c30b880-3325-11eb-8828-898b8a8f72c5.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 12755. Press the Button and then tap the BoxView that has just changed color. If a dialog appears, the test has passed.

### PR Checklist ###
